### PR TITLE
Playground demonstrating unecessary [unowned self] for lazy vars

### PR DESCRIPTION
**Pull request by noahsark769**
@gsheld here's the test you wanted :)

### CODE REVIEW

File: swift/unowned-self-in-lazy-property.playground/Contents.swift [hunk 1]
```diff
@@ -0,0 +1,44 @@
+/// This playground checks to make sure that you don't need to do [unowned self] in a lazy var
```
Comment (gengelha) [inline swift/unowned-self-in-lazy-property.playground/Contents.swift:1]: @noah Completely forgot about your PR! Mind updating this header to match the templates that have already been used?

Something like...

```
/**
 * GOAL:
 *    To determine if multiple calls to NSLayoutConstraint.activateConstraints() lead to a duplication of NSLayoutConstraint
 *    instances associated with their respective views.
 *
 * FINDINGS: 
 *    Multiple calls to NSLayoutConstraint.activateConstraints() does NOT lead to a duplication of NSLayoutConstraint instances
 *    associated with their respective views.
 *
 * Version: Apple Swift version 2.2 (swiftlang-703.0.18.5 clang-703.0.31)
 * Date:    4-28-2016
 * Author:  Grant Sheldon
 */
```

File: swift/unowned-self-in-lazy-property.playground/Contents.swift [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+import Foundation
+
+class MyClass {
+  deinit {
+    print("MyClass dealloc")
+  }
+}
+
+class MyParentClass {
```
Comment (gengelha) [inline swift/unowned-self-in-lazy-property.playground/Contents.swift]: Its strange to refer to this as a "Parent" class without it inheriting from `MyClass`.

Comment (noahsark769) [reply swift/unowned-self-in-lazy-property.playground/Contents.swift]: True, MyContainerClass would probably be better

File: swift/unowned-self-in-lazy-property.playground/Contents.swift (hunk 1 above)
Comment (noahsark769) [reply swift/unowned-self-in-lazy-property.playground/Contents.swift:1]: Done!

**Review by gengelha — APPROVED** [review]

**Pull request merged**